Python/bytecodes.c

```diff
@@ -4325,7 +4325,10 @@ dummy_func(
                 if (tuple == NULL) {
                     ERROR_NO_POP();
                 }
-                PyStackRef_SET(callargs_st, PyStackRef_FromPyObjectSteal(tuple));
+                _PyStackRef *temp_dst_ptr = &callargs_st;
+                _PyStackRef temp_old_dst = *temp_dst_ptr;
+                *temp_dst_ptr = PyStackRef_FromPyObjectSteal(tuple);
```

Comment:
Why create this reference just to close it?

Reply:
It's the SETREF pattern:
```
/* Safely decref `dst` and set `dst` to `src`.
 *
 * As in case of Py_CLEAR "the obvious" code can be deadly:
 *
 *     Py_DECREF(dst);
 *     dst = src;
 *
 * The safe way is:
 *
 *      Py_SETREF(dst, src);
 *
 * That arranges to set `dst` to `src` _before_ decref'ing, so that any code
 * triggered as a side-effect of `dst` getting torn down no longer believes
 * `dst` points to a valid object.
```

Reply:
The destination is a local variable. There is no "side-effect of `dst` getting torn down"